src/main/java/duke/myExceptions/InvalidInput.java

```diff
@@ -0,0 +1,4 @@
+package duke.myExceptions;
+
+public class InvalidInput extends Exception{
```

Comment:
It seems this class does not do anything different from its superclass, which is `Exception`. You could consider leaving out redundant classes or update it with the new information. This comment applies to the other newly created exception classes.